diff --git a/mklove/modules/configure.base b/mklove/modules/configure.base
index 77cee61a..d5a85ee7 100644
--- a/mklove/modules/configure.base
+++ b/mklove/modules/configure.base
@@ -2120,6 +2120,11 @@ function mkl_module_load {
     local save_MKL_MODULE=$MKL_MODULE
     MKL_MODULE=$modname
 
+    if [[ $fname = "configure.self" ]]; then
+        echo "Renaming configure.self -> ./configure.self"
+        fname="./configure.self"
+    fi
+
     mkl_dbg "Loading module $modname (required by $cmod) from $fname"
 
     # Source module file (positional arguments are available to module)
